roles/docker/tasks/main.yml

```diff
@@ -118,7 +124,7 @@
     force: "{{item.force|default(omit)}}"
     conf_file: "{{item.yum_conf|default(omit)}}"
     state: present
-    update_cache: yes
+    update_cache: "{{ omit if ansible_distribution == 'Fedora' else True }}"
```

Comment:
why?

Reply:
Ansible dnf module does not support "update_cache"

Reply:
is there a why to test it's the dnf module? 
 `omit if xxxx == 'dnf' `   I 'd like to avoid too specific os conditions

Reply:
This is a good idea.
It does feel much less "hacky"